src/main/play-doc/operation/Install.md

```diff
@@ -1235,7 +1245,7 @@ frontend conductr-haproxy-test
   mode http
   monitor-uri /test
 
-{{#eachAcls bundles defaultHttpPort=9000}}
+{{#eachAcls bundles defaultHttpPort=80}}
```

Comment:
Good point. Bundles such as project-doc use 80, so I mapped it.

Seems I will just add 9000 to docker command and revert to 9000

On Wed, Aug 24, 2016, 20:50 Christopher Hunt <notifications@github.com>
wrote:

> In src/main/play-doc/operation/Install.md
> <https://github.com/typesafehub/conductr-doc/pull/255#discussion_r76176504>
> :
>
> > @@ -1235,7 +1245,7 @@ frontend conductr-haproxy-test
> >    mode http
> >    monitor-uri /test
> >
> > -{{#eachAcls bundles defaultHttpPort=9000}}
> > +{{#eachAcls bundles defaultHttpPort=80}}
>
> Also port 80 might clash with lb-marathon
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/typesafehub/conductr-doc/pull/255/files/f123f9fe47e4e16ba4df1cd587fb2e9fcc4b8270#r76176504>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAwCo97BegxEuv9nNasD3vuPNd7WJA3pks5qjRD3gaJpZM4JslBg>
> .
>